readthedocs-theme/templates/includes/pagination.html

```diff
@@ -0,0 +1,20 @@
+{% if DEFAULT_PAGINATION %}
+  {% set first_page = articles_paginator.page(1) %}
+  {% set last_page = articles_paginator.page(articles_paginator.num_pages) %}
```

Comment:
It looks like we set up to do a numeric pagination, but then don't use the variables. Is numeric pagination something we want here?

https://semantic-ui.com/collections/menu.html#pagination

Reply:
Removed line 2 and 3 in commit https://github.com/readthedocs/site-community/pull/64/commits/82c08d2a5812e2409503d30bbf018347e84ba5ba.